api/controllers/tasks.py

```diff
@@ -120,6 +125,23 @@ def process_proposal(credentials, tpid):
             last_updated=db.sql.func.now(),
         )  # Annotation config is sentiment example.
 
+        emailReceiver = tp_creator_email
+        cc = "no-reply@dynabench.org"
+        subject = "Dynabench Task Approval Mail"
+
+        try:
+            mail.send(
+                server="",
+                config=config,
+                contacts=[emailReceiver],
+                cc_contact=cc,
+                template_name="templates/task_approval.txt",
+                subject=subject,
+            )
+            print("Email sent successfully!")
+        except smtplib.SMTPException:
```

Comment:
if I'm not wrong there's no need for a try-except block if it only wraps `mail.send()` since the `mail.send()` function already catches any exceptions.

Reply:
this also means that there's no need to `import smtplib` in line 6